deploy/ipfs-entrypoint.sh

```diff
@@ -0,0 +1,57 @@
+#!/bin/sh
+set -e
+user=ipfs
+repo="$IPFS_PATH"
+
+if [ `id -u` -eq 0 ]; then
+  echo "Changing user to $user"
+  # ensure folder is writable
+  su-exec "$user" test -w "$repo" || chown -R -- "$user" "$repo"
+  # restart script with new privileges
+  exec su-exec "$user" "$0" "$@"
+fi
+
+# 2nd invocation with regular user
+ipfs version
+
+if [ -e "$repo/config" ]; then
+  echo "Found IPFS fs-repo at $repo"
+else
+  case "$IPFS_PROFILE" in
```

Comment:
Lets default to the `server` profile here.

Reply:
I think we should optimize for production deployment using the hints from here: https://cluster.ipfs.io/documentation/deployment/setup/

Reply:
Done